internal/apiserver/draft_test.go

```diff
@@ -45,14 +47,55 @@ func TestGetMyDrafts(t *testing.T) {
 	assert.ElementsMatch(t, drafts, responseData)
 }
 
-func TestDraft(t *testing.T) {
+func TestDraftsExpiration(t *testing.T) {
 	t.Cleanup(testutil.CleanDb(ctx, t, a.db.Connector))
 	t.Cleanup(testutil.CleanCache(ctx, t, a.cache))
 	t.Cleanup(testutil.CleanStorage(t, a.storage))
 
 	user := testutil.CreateUser(ctx, t, a.db.Connector, "")
 	sessionToken := testutil.CreateToken(ctx, t, a.cache, user)
 
+	id1 := ulid.New(time.Now()).String()
+	id2 := ulid.New(time.Now().Add(-24 * time.Hour)).String()
+	draft1 := testutil.CreateDraft(ctx, t, a.storage, a.cache, user.Id, "d1", "", id1)
+	draft2 := testutil.CreateDraft(ctx, t, a.storage, a.cache, user.Id, "d2", "", id2)
+
+	req, err := http.NewRequest("GET", "/drafts", nil)
+	if err != nil {
+		t.Error(err)
+	}
+
+	response := testutil.ExecuteAuthRequest(a, req, sessionToken)
+	assert.Equal(t, http.StatusOK, response.Code)
+	var responseData []*entity.Draft
+	if err = json.Unmarshal(response.Body.Bytes(), &responseData); err != nil {
+		t.Error(err)
+	}
+
+	draft1.Data = nil
+	assert.ElementsMatch(t, []*entity.Draft{draft1}, responseData)
```

Comment:
maybe we want to assert order as well (not only elements).
Now it is only 1 element array, but we can add some to this test in future